ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+The following is an template for members to use whenever creating an issue for members or the open-source community. Each section should be filled out unless stated as optional. For problem type, state whether the issue is targetting an error/bug, documentation fix, need for additional information, etc. If you have any other questions about creating or contributing, please checkout the *"how to contribute"* Github issue [here](https://github.com/bitprj "here") (link coming soon).
+
+**Template:**
+
+---
+
+
+### Issue Name:
+
+### Problem:
```

Comment:
I think under each "heading" we could add a line about what the naming convention/what should be under each one. For example, is a problem being "slack apps docs suck" probably isn't what we want.

Reply:
Good idea. I added additional "headings" under each segment @lazyplatypus.